cmd/containerd-shim/main_unix.go

```diff
@@ -60,6 +60,7 @@ func main() {
 	if debugFlag {
 		logrus.SetLevel(logrus.DebugLevel)
 	}
+	runtime.GOMAXPROCS(2)
```

Comment:
It would be nice to still allow users to override this if they want. How about

```go
if os.Getenv("GOMAXPROCS") == "" {
    runtime.GOMAXPROCS(2)
}
```

Reply:
I think this is a good idea to add with this change.